iac-extensions/kubernetes/src/main/java/org/sonar/iac/kubernetes/visitors/LocationShifter.java

```diff
@@ -136,8 +142,9 @@ public static TextRange computeShiftedLocation(HelmInputFileContext ctx, TextRan
   }
 
   /**
-   * Adjust given {@link TextRange} to Helm template Value Path location.
-   * If no Value Path is found at provided {@link TextRange} the original {@link TextRange} is returned. <p/>
+   * Adjust given {@link TextRange} to Helm template Value Path location (e.g. {@code .Values.foo.bar} ) or
+   * include function parameter, or toYaml function.
```

Comment:
```suggestion
   * include function parameter, or toYaml function, or another element to be highlightied.
```
Not to have to edit javadocs each time we enhance it :)